client/a8c-for-agencies/components/pending-payment-notification/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { Button } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import LayoutBanner from 'calypso/a8c-for-agencies/components/layout/banner';
+import { useDispatch, useSelector } from 'calypso/state';
+import { getActiveAgency } from 'calypso/state/a8c-for-agencies/agency/selectors';
+import { setPreference } from 'calypso/state/preferences/actions';
+import { getPreference } from 'calypso/state/preferences/selectors';
+import { CONTACT_URL_HASH_FRAGMENT } from '../a4a-contact-support-widget';
+import { A4A_INVOICES_LINK } from '../sidebar-menu/lib/constants';
+
+import './style.scss';
+
+const PENDING_PAYMENT_NOTIFICATION_DISMISS_PREFERENCE = 'pending-payment-notification-dismissed';
+
+export default function PendingPaymentNotification() {
+	const dispatch = useDispatch();
+	const agency = useSelector( getActiveAgency );
+
+	const isDismissed = useSelector( ( state ) =>
+		getPreference( state, PENDING_PAYMENT_NOTIFICATION_DISMISS_PREFERENCE )
+	);
```

Comment:
Should we use a preference if we will keep displaying the notification after every page refresh? I wonder if using React `useState` is enough, considering that.

Reply:
The reason I'm using the preference here is the same component is used on many pages, and it reappears when visiting the other pages if we use `useState` here. To fix that, we are saving it temporarily using `setPreference`

Reply:
That makes sense. Thanks for explaining that.